HW_1/task_2.py

```diff
@@ -0,0 +1,8 @@
+"""
+Докажите, что выражение ¬(X ⋁ Y ⋁ Z) = ¬X ⋀ ¬Y ⋀ ¬Z истинно для всех значений предикат.
+
+Вывод: единственное значение типа bool (True либо False)
+"""
+
+x, y, z = 0, 0, 0
+print(not(x or y or z) == (not(x) and not(y) and not(z)))
```

Comment:
К сожалению, ваше решение не полное. Вы берете только по одному значению каждого предиката, а надо проверить для всех значений.
Лишние скобки:
```suggestion
print(not(x or y or z) == (not x and not y and not z))
```